### ISSUE TITLE
New build issue on Debian

### ISSUE THREAD

**Issue by erikd** [vshabanov/HsOpenSSL#21]
Building git HEAD (630c97d6e4f5ed) with ghc 8.0.2:
```
Building HsOpenSSL-0.11.4.4...
Preprocessing library HsOpenSSL-0.11.4.4...
cbits/HsOpenSSL.c: In function ‘HsOpenSSL_HMAC_CTX_free’:

cbits/HsOpenSSL.c:86:12: error:
     error: too few arguments to function ‘HMAC_CTX_free’
         return HMAC_CTX_free();
                ^~~~~~~~~~~~~

In file included from cbits/HsOpenSSL.h:9:0: error:
    0,
                     from cbits/HsOpenSSL.c:2:

/usr/include/openssl/hmac.h:26:6: error:
     note: declared here
     void HMAC_CTX_free(HMAC_CTX *ctx);
          ^~~~~~~~~~~~~

cbits/HsOpenSSL.c:86:12: error:
     warning: ‘return’ with a value, in function returning void
         return HMAC_CTX_free();
                ^~~~~~~~~~~~~

cbits/HsOpenSSL.c:84:6: error:
     note: declared here
     void HsOpenSSL_HMAC_CTX_free(HMAC_CTX *ctx) {
          ^~~~~~~~~~~~~~~~~~~~~~~
`gcc' failed in phase `C Compiler'. (Exit code: 1)
Process failed: cabal build -j --ghc-options=-Werror (exit code: 1)
```

**Comment by vshabanov** [comment]
Thank you for reporting this. Fixed and uploaded on Hackage (HsOpenSSL-0.11.4.5).

**Comment by erikd** [comment]
:100: Works as expected!
